src/Common/Http/Client/Cache.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\Common\Http\Client;
+
+class Cache extends Client
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * @var \Doctrine\Common\Cache\Cache
```

Comment:
Boo.. Why not use [PSR-16](https://www.php-fig.org/psr/psr-16/) for better inter-op?

Reply:
I am okey with it, done in

https://github.com/SocialConnect/auth/pull/84/commits/e0ce912304d6c4e3c04a15160ac0e77528c08870
https://github.com/SocialConnect/auth/pull/84/commits/c60d23098380938967271ea0784f86850da081ac